### PR TITLE
br: add restriction for incremental restore

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add some restriction of incremental backup.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by BornChanger — APPROVED** [review]

File: br/br-incremental-guide.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ TiDB 集群增量数据包含在某个时间段的起始和结束两个快照的
 >
 > 当前该功能已经停止开发迭代，推荐你选择[日志备份与恢复功能](/br/br-pitr-guide.md)代替。
 
+## 使用限制
+
+由于增量备份的恢复过程中，依赖于备份时间点的库表快照进行增量 DDL 的过滤，因此对于增量备份过程中删除的表，在恢复后可能仍然存在，需要手动删除。
+增量备份尚不支持批量重命名表这一类型的 DDL，如果选定的增量备份范围内发生了批量重命名表的操作，那么有几率造成恢复失败。建议在批量重命名后进行一次全量备份，并在恢复中用最新的全量备份替代增量数据。
```
Comment (lilin90) [inline br/br-incremental-guide.md:17]:
```suggestion
由于增量备份的恢复依赖于备份时间点的库表快照来过滤增量 DDL，因此对于增量备份过程中删除的表，在恢复后可能仍然存在，需要手动删除。

增量备份尚不支持批量重命名这一类型的 DDL，如果在选定的增量备份范围内发生了批量重命名表的操作，那么有可能造成恢复失败。建议在批量重命名增量 DDL 后进行一次全量备份，并在恢复时使用最新的全量备份替代增量数据。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by hfxsd — APPROVED** [review]

File: br/br-incremental-guide.md [hunk 2]
```diff
@@ -11,6 +11,12 @@ TiDB 集群增量数据包含在某个时间段的起始和结束两个快照的
 >
 > 当前该功能已经停止开发迭代，推荐你选择[日志备份与恢复功能](/br/br-pitr-guide.md)代替。
 
+## 使用限制
+
+由于增量备份的恢复依赖于备份时间点的库表快照来过滤增量 DDL，因此对于增量备份过程中删除的表，在恢复后可能仍然存在，需要手动删除。
+
+增量备份尚不支持批量重命名增量 DDL，如果在选定的增量备份范围内发生了批量重命名表的操作，那么有可能造成恢复失败。建议在批量重命名增量 DDL 后进行一次全量备份，并在恢复时使用最新的全量备份替代增量数据。
```
Comment (BornChanger) [inline br/br-incremental-guide.md:18]:
```suggestion
增量备份尚不支持批量重命名表操作，如果选定的增量备份范围内发生了批量重命名表的操作，那么有可能造成恢复失败。建议在批量重命名表后进行一次全量备份，并在恢复时使用最新的全量备份替代增量数据。
```

File: br/br-incremental-guide.md [hunk 3]
```diff
@@ -11,6 +11,12 @@ TiDB 集群增量数据包含在某个时间段的起始和结束两个快照的
 >
 > 当前该功能已经停止开发迭代，推荐你选择[日志备份与恢复功能](/br/br-pitr-guide.md)代替。
 
+## 使用限制
+
+由于增量备份的恢复依赖于备份时间点的库表快照来过滤增量 DDL，因此对于增量备份过程中删除的表，在恢复后可能仍然存在，需要手动删除。
+
+增量备份尚不支持批量重命名表操作，如果选定的增量备份范围内发生了批量重命名表的操作，那么有可能造成恢复失败。建议在批量重命名表后进行一次全量备份，并在恢复时使用最新的全量备份替代增量数据。
```
Comment (lilin90) [inline br/br-incremental-guide.md:18]:
```suggestion
增量备份尚不支持批量重命名表的操作。如果选定的增量备份范围内发生了批量重命名表的操作，则有可能造成恢复失败。建议在批量重命名表后进行一次全量备份，并在恢复时使用最新的全量备份替代增量数据。
```

File: br/br-incremental-guide.md [hunk 4]
```diff
@@ -11,6 +11,12 @@ TiDB 集群增量数据包含在某个时间段的起始和结束两个快照的
 >
 > 当前该功能已经停止开发迭代，推荐你选择[日志备份与恢复功能](/br/br-pitr-guide.md)代替。
 
+## 使用限制
+
+由于增量备份的恢复依赖于备份时间点的库表快照来过滤增量 DDL，因此对于增量备份过程中删除的表，在恢复后可能仍然存在，需要手动删除。
+
+增量备份尚不支持批量重命名表的操作。如果选定的增量备份范围内发生了批量重命名表的操作，则有可能造成恢复失败。建议在批量重命名表后进行一次全量备份，并在恢复时使用最新的全量备份替代增量数据。
```
Comment (lilin90) [inline br/br-incremental-guide.md:18]:
```suggestion
增量备份尚不支持批量重命名表的操作。如果在增量备份过程中发生了批量重命名表的操作，则有可能造成恢复失败。建议在批量重命名表后进行一次全量备份，并在恢复时使用最新的全量备份替代增量数据。
```

**Comment by lilin90** [comment]
After confirming with @BornChanger, we won't apply changes in this PR to v6.1 docs.

**Pull request merged**